source/beamline/AMPVControl.cpp

```diff
@@ -89,25 +92,40 @@ void AMReadOnlyPVControl::onReadPVError(int errorCode) {
 	}
 }
 
-
-void AMReadOnlyPVControl::onLowLimitPVValueChanged(double newLowLimit)
+void AMReadOnlyPVControl::setLowLimitValue(double newLowLimit)
 {
-	lowLimitValue_ = newLowLimit;
-	emit minimumValueChanged(lowLimitValue_);
+	if (lowLimitValue_ != newLowLimit) {
```

Comment:
There may be a case when we would want the signals to be emitted here but the new value fails this if-check--when the actual limit is equivalent to the default value. It may make sense to remove these if-checks from AMReadOnlyPVControl::setLowLimitValue() and AMReadOnlyPVControl::setHighLimitValue(). Then the signals would be emitted every time a value is set.